### PR TITLE
Transform com_finder to ES6 and remove inline script [ Part 2 ] 

### PR DESCRIPTION
**Pull request by anuragteapot**
Pull Request for Issue 
1.  #8 
2.  #55

### CODE REVIEW

**Review by C-Lodder — APPROVED** [review]

File: media/com_finder/js/maps.es6.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'map.delete') {
+        if (window.confirm(Joomla.JText._('COM_FINDER_MAPS_CONFIRM_DELETE_PROMPT'))) {
+          Joomla.submitform(pressbutton);
+        } else {
+          return false;
+        }
+      }
+      Joomla.submitform(pressbutton);
+      return true;
```
Comment (dneukirchen) [inline media/com_finder/js/maps.es6.js]: Same as in https://github.com/joomla-projects/joomla-es6/pull/90#pullrequestreview-102567078

File: media/com_finder/js/maps.es6.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Joomla = window.Joomla || {};
+
+(() => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'map.delete') {
+        if (window.confirm(Joomla.JText._('COM_FINDER_MAPS_CONFIRM_DELETE_PROMPT'))) {
```
[the remark this answers is not in the document]
Comment (dneukirchen) [reply media/com_finder/js/maps.es6.js]: please add a todo

**Review by dneukirchen — CHANGES_REQUESTED** [review]

**Comment by anuragteapot** [comment]
@dneukirchen done!

**Review by dneukirchen — APPROVED** [review]

**Comment by dneukirchen** [comment]
thx @Anu1601CS

**Pull request merged**